services/src/main/java/org/fao/geonet/api/es/EsHTTPProxy.java

```diff
@@ -275,8 +275,8 @@ private static boolean hasOperation(ObjectNode doc, ReservedGroup group, Reserve
 
 
     @io.swagger.v3.oas.annotations.Operation(
-        summary = "Search endpoint",
-        description = "See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html for search parameters details.")
+        summary = "Execute a search query and get back search hits that match the query.",
+        description = "The search API execute a search query (q query string or JSON request body). For more information see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html for search parameters, and https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html Query DSL.")
```

Comment:
It is unclear to me if q query string is supported?

Reply:
What do you mean? This end-point supports `POST` and accepts a body with an Elasticsearch query